src/lib.rs

```diff
@@ -291,8 +337,12 @@ where
         Ok(())
     }
 
-    pub fn set_dio0_tx_done(&mut self) -> Result<(), Error<E, CS::Error, RESET::Error>> {
```

Comment:
I would keep this method for backwards compatibility and use `set_dio` internally. Also I would mark it with [deprecated attribue](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute) possibly with a message directing user to use `set_dio` directly.

Reply:
ok .. set_dio0_tx is back for backward compatibility